src/test/java/com/SAS/soccer_association_system/SoccerAssociationSystemApplicationTests.java

```diff
@@ -10,7 +10,7 @@
 
 	public static void main(String[] args) {
 
-		SpringApplication.run(SoccerAssociationSystemApplication.class, args);
+		//SpringApplication.run(SoccerAssociationSystemApplication.class, args);
 
 
 
```

Comment:
please stop comment out things that you commit, only commit code that is neccesry, keep the code that you are not using on your local reposetory. in the featurue.